### PR TITLE
doc: add Google Group instructions to onboarding

### PR DESCRIPTION
**Pull request by mmarchini**
Ref: https://github.com/nodejs/TSC/issues/1133#issuecomment-1076786899

### CODE REVIEW

**Review by cjihrig — APPROVED** [review]

File: ONBOARDING.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ invalid and are superseded by the TSC Charter.
 * Invite them to the HackerOne [Node.js team](https://hackerone.com/nodejs/team_members).
   Have them turn on two-factor authentication after they have joined.
 * Add them to the `tsc` and `crypto-export` mailing lists.
+* Add them to the [nodejs-tsc](https://groups.google.com/g/nodejs-tsc) Google Group as owners, so they receive Calendar invites for TSC meetings.
```
Comment (richardlau) [inline ONBOARDING.md:22]: Should I be able to access this link? I get a "content unavailable" error message.

Comment (richardlau) [reply ONBOARDING.md:22]: Ah never mind. Just saw the invite and the link works after accepting.

**Review by richardlau — APPROVED** [review]

**Review by tniessen — APPROVED** [review]

File: ONBOARDING.md [hunk 2]
```diff
@@ -35,6 +36,7 @@ invalid and are superseded by the TSC Charter.
 * Remove them from the `@pkgjs` org unless they are members for a
   reason other than TSC membership.
 * Remove them from the `tsc` and `crypto-export` mailing lists.
+* Add them to the [nodejs-tsc](https://groups.google.com/g/nodejs-tsc) Google Group so they receive Calendar invites for TSC meetings.
```
Comment (Trott) [inline ONBOARDING.md:39]:
```suggestion
* Remove them from the [nodejs-tsc](https://groups.google.com/g/nodejs-tsc) Google Group.
```

Comment (tniessen) [reply ONBOARDING.md:39]: Good catch!

**Review by RaisinTen — APPROVED** [review]
LGTM with @Trott's suggestion

**Review by Trott — APPROVED** [review]

**Pull request merged**